packages/common/driver-definitions/package.json

```diff
@@ -63,35 +63,8 @@
 	},
 	"typeValidation": {
 		"broken": {
-			"TypeAliasDeclaration_DriverError": {
-				"backCompat": false
-			},
-			"EnumDeclaration_DriverErrorType": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IAuthorizationError": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IDriverBasicError": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IDriverErrorBase": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IGenericNetworkError": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_ILocationRedirectionError": {
-				"backCompat": false
-			},
-			"InterfaceDeclaration_IThrottlingWarning": {
-				"backCompat": false
-			},
 			"VariableDeclaration_DriverErrorTypes": {
 				"forwardCompat": false
-			},
-			"TypeAliasDeclaration_DriverErrorTypes": {
-				"backCompat": false
 			}
```

Comment:
First instance of 3: Type tests that now pass due to the more permissive comparison of string-like and number-like.

In this case, the new permissive TypeOnly check is allowing an enum-like intersection of string literals (i.e., 'genericError' | 'networkError' | ...) as just a 'string'.